osquery/killswitch/plugins/killswitch_filesystem.h

```diff
@@ -0,0 +1,20 @@
+#include "osquery/killswitch/plugins/killswitch_json.h"
+#include <string>
+
+namespace osquery {
+
+class KillswitchFilesystem : public KillswitchJSON {
+ public:
+  KillswitchFilesystem();
+  KillswitchFilesystem(const std::string& conf_path);
+
+ protected:
```

Comment:
Why you don't want do make it just public?

Reply:
I don't want it to be abused. The only way I want it to be used is for the refresh. and protected is enough for it. If anyone finds any legit use case for making it public, I don't mind ofc.